source/NVDAObjects/UIA/__init__.py

```diff
@@ -2006,6 +2007,27 @@ def _get_positionInfo(self):
 	def scrollIntoView(self):
 		pass
 
+	def isDescendantOf(self, obj):

```

Comment:
add type annotations

```suggestion
	def isDescendantOf(self, obj: "NVDAObjects.NVDAObject") -> bool:
```